redisinsight/ui/src/pages/autodiscover-cloud/redis-cloud-databases/RedisCloudDatabases/RedisCloudDatabases.tsx

```diff
@@ -230,26 +213,25 @@ const RedisCloudDatabasesPage = ({
                 desc: false,
               },
             ]}
-            onRowClick={selectionValue.onSelectionChange}
           />
-          {!items.length && <Text>{message}</Text>}
-        </div>
+          {!items.length && <Text size="S">{message}</Text>}
+        </DatabaseWrapper>
       </div>
-      <FlexItem padding={4}>
-        <Row justify="between" gap="m">
+      <Footer padding={4}>
```

Comment:
I guess we don't have a design for reference, but I saw that the footer is "slightly" misaligned
- see the border of the main content container is not in line with the border in the side nav
- and if we fix that, the action button in the footer will have proper padding

PS:> Also, the border on the right is going outside of the container

<img width="2598" height="1652" alt="Image" src="https://github.com/user-attachments/assets/5f250cde-22ed-452a-b77e-c417ebb33448" />